domain/src/main/java/ru/startandroid/domain/models/Condition_.kt

```diff
@@ -0,0 +1,18 @@
+package ru.startandroid.domain.models
```

Comment:
Do we need this file? Looks like Condition.kt but with underscore.

Reply:
According to Json hierarchy, we have two objects within Json data - Condition.kt is external and Condition_.kt is the inner one. I decided not to rename them and use the original naming.

Reply:
Let's use only Condition and fix it in Day class to:
 var condition: Condition? = null